esp-backtrace/src/lib.rs

```diff
@@ -1,6 +1,25 @@
 #![allow(rustdoc::bare_urls, unused_macros)]
 #![cfg_attr(target_arch = "xtensa", feature(asm_experimental_arch))]
-#![doc = include_str!("../README.md")]
+//! Supports the ESP32, ESP32-C2/C3/C6, ESP32-H2, ESP32-P4, and ESP32-S2/S3. Optional exception and
+//! panic handlers are included, both of which can be enabled via their respective features.
+//!
+//! Please note that when targeting a RISC-V device, you **need** to force frame pointers (i.e.
```

Comment:
Let's cfg this comment on `target_arch = "riscv32"` and reword it a bit so it makes sense